zillow_crawler/main.py

```diff
@@ -40,6 +44,7 @@ def main():
 
         for listing in listings:
             cons.print(listing.address)
+            total_houses += 1
```

Comment:
I would use Python's `len()` functions on the `listings` list to get `total_houses`.  Some would call it more Pythonic since `len()` is built into the language.

Reply:
I call it "mature coding". All set.

It continues to crawl zero pages, so I can't run the code and see if works